java/code/src/com/suse/manager/webui/controllers/utils/RegularMinionBootstrapper.java

```diff
@@ -113,4 +124,49 @@ else if (weRejectedIt) {
         LOG.info("Minion bootstrap success: " + result.isSuccess());
         return result;
     }
+
+    /**
+     * We want to cleanup the authorized ssh key, which was copied by Ansible and was used for authentication of the
+     * primal bootstrap call.
+     *
+     * Since we don't have the minion id of the bootstrap minion yet, we need to use Ansible for removing the authorized
+     * key.
```

Comment:
Not sure if we should do it this way.

1. what if they want to do salt-ssh management? AFAIK we need this key where it is.
2. We could also write a salt state which does the cleanup. As the client is onboarded we could do it there as well

Or did I miss something?

Reply:
> 1. what if they want to do salt-ssh management? AFAIK we need this key where it is.

Yes, that's why this method is overriden as a no-op in the `SSHMinionBootstrapper`.

> 2. We could also write a salt state which does the cleanup. As the client is onboarded we could do it there as well

This could also do the thing and maybe it'd be more straightforward.

Reply:
I moved the cleanup to salt bootstrap state. I'm going to test it now.